collections.json

```diff
@@ -1,4 +1,9 @@
 [
+  {
+    "collection": "ENS: Ethereum Name Service",
+    "address": "0xfbB0c2E86B2A26EdD8440d495c982Ad0BeCd84A6",
```

Comment:
```
0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85
```
This should be the contract that you're adding (underlying NFT - ENS name service).

Reply:
Okay I have updated this in the new commit. Thank you for the clarification!